src/postgres/commands/createPostgresFunctionQuery.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizard } from "vscode-azureextensionui";
+import { nonNullProp } from "../../utils/nonNull";
+import * as vscodeUtil from '../../utils/vscodeUtils';
+import { PostgresDatabaseTreeItem } from "../tree/PostgresDatabaseTreeItem";
+import { connectPostgresDatabase } from "./connectPostgresDatabase";
+import { FunctionQueryCreateStep } from "./FunctionQueryWizard/FunctionQueryCreateStep";
+import { FunctionQueryNameStep } from "./FunctionQueryWizard/FunctionQueryNameStep";
+import { FunctionQueryReturnTypeStep } from "./FunctionQueryWizard/FunctionQueryReturnTypeStep";
+import { IPostgresFunctionQueryWizardContext } from "./FunctionQueryWizard/IPostgresFunctionQueryWizardContext";
+
+export async function createPostgresFunctionQuery(wizardContext: IPostgresFunctionQueryWizardContext, treeItem?: PostgresDatabaseTreeItem): Promise<void> {
```

Comment:
You should do `context: IActionContext` as the first parameter and create the wizardContext below like this:
```
const wizardContext: IPostgresFunctionQueryWizardContext = context;
```

It might seem silly, but it has better type validation. For example, [here](https://github.com/microsoft/vscode-azurefunctions/blob/9d9a73014c5edb17db523ba907ab017ff005dabb/src/commands/createFunction/createFunction.ts#L45)'s an example in the functions repo where several properties like `projectPath` are required on the wizardContext (aka they have to be set before the wizard starts). If I accidentally misspell projectPath like this:
```
const wizardContext: IFunctionWizardContext = Object.assign(context, { projectOOPSPath ... });
```

It would show this error:
> Property 'projectPath' is missing in type 'IActionContext & { projectOOPSPath: string; workspacePath: string; workspaceFolder: WorkspaceFolder | undefined; version: FuncVersion; language: ProjectLanguage; functionName: string | undefined; }' but required in type 'IFunctionWizardContext'.ts(2741)

NOTE: An alternative fix is to turn on `strictFunctionTypes` in tsconfig, but that's a lot more work affecting the whole repo